help/destinations/catalog/crm/salesforce.md

```diff
@@ -137,41 +150,58 @@ When you are finished providing details for your destination connection, select
 
 Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
 
 ### Mapping considerations and example {#mapping-considerations-example}
 
-To correctly send your audience data from Adobe Experience Platform to the Salesforce CRM destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the Salesforce CRM destination fields, follow these steps:
-
-1. In the Mapping step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Salesforce CRM] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the [!DNL Salesforce CRM] destination fields, follow these steps:
 
-    ![Add new mapping](../../assets/catalog/crm/salesforce/add-new-mapping.png)
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+![Platform UI screenshot example for Add new mapping.](../../assets/catalog/crm/salesforce/add-new-mapping.png)
 
-1. In the select source field window, when selecting the source field choose the **[!UICONTROL Select attributes]** category and add the mappings desired.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select identity namespace]** category and select `crmID`.
+![Platform UI screenshot example for Source mapping.](../../assets/catalog/crm/salesforce/source-mapping.png)
 
-    ![Source mapping](../../assets/catalog/crm/salesforce/source-mapping.png)
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** category and select `SalesforceId`.
+    * **[!UICONTROL Select identity namespace]**: select this option to map your source field to an identity namespace from the list.
+    ![Platform UI screenshot showing Target mapping for SalesforceId.](../../assets/catalog/crm/salesforce/target-mapping-salesforceid.png)
+    * Add the following mapping between your XDM profile schema and your [!DNL Salesforce CRM] instance:
 
-1. In the select target field window, select the target field and choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
+    |XDM Profile Schema|[!DNL Salesforce CRM] Instance| Mandatory |
+    |---|---|---|
+    |`crmID`|`SalesforceId`|Yes|
```

Comment:
`crmID` or any other identity, correct?

Reply:
Updated.

Reply:
Same as just above. The table makes it look like crmID is the only accepted identity on the source side. Is that the case?

Reply:
Yes.
FYI @vgiurgiu at the time of marking as resolved had to yet commit the changes, multiple threads and so as to not lose track had marked as updated here.